frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-overview/pipeline-overview-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { PIPELINE_RUN_AUTO_START_FAILED } from '@console/dev-console/src/components/topology/redux/const';
+
+export const getNotStartedPipelines = () => {
+  return JSON.parse(sessionStorage.getItem(PIPELINE_RUN_AUTO_START_FAILED) ?? '[]');
+};
+export const isPipelineNotStarted = (pipelineName: string): boolean => {
+  return getNotStartedPipelines().includes(pipelineName);
+};
+
+export const removePipelineNotStarted = (pipelineName: string): void => {
+  const newList = getNotStartedPipelines().filter((pName) => pName !== pipelineName);
```

Comment:
Based on my last(last) comment... need namespace here.

Reply:
@andrewballantyne Updated the code to store/remove a namespaced session object.